wnfs/src/private/node.rs

```diff
@@ -607,6 +607,21 @@ impl PrivateNodeHeader {
     }
 }
 
+impl Debug for PrivateNodeHeader {
```

Comment:
Do we want to keep custom `Debug` implementations?
I'm really not against it. I've previously had a bunch of these implementations for a couple of our structs in one of my PRs, but removed them since I felt like customizing them may be less ... idiomatic?

Reply:
We want to keep them. The inumbers are easier read formatted as a single byte string esp. when pretty printing `{:#?}` is applied.